numba/cuda/tests/cudapy/test_cooperative_groups.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import print_function
+
+import numpy as np
+
+from numba import cuda, config
+from numba.cuda.testing import unittest, SerialMixin, skip_on_cudasim
+
+
+def this_grid(A):
+    A[0] = cuda.this_grid()
+
+
+def sync_group(A):
+    g = cuda.this_grid()
+    cuda.sync_group(g)
+    A[0] = g
+
+
+if not config.ENABLE_CUDASIM:
+    from numba.cuda.cudadrv.libs import find_lib
+
+    LINK = find_lib(
```

Comment:
This is failing on Mac.

Reply:
Indeed :) Since the `cudatoolkit` package doesn't install it ([I've raised an issue](numba/conda-recipe-cudatoolkit#19)) it's not really clear where to look for it on a mac.

Reply:
Issue link is https://github.com/numba/conda-recipe-cudatoolkit/issues/19